src/main/java/edu/kis/powp/jobs2d/features/CommandsFeature.java

```diff
@@ -1,25 +1,39 @@
 package edu.kis.powp.jobs2d.features;
 
+import edu.kis.powp.appbase.Application;
 import edu.kis.powp.jobs2d.command.manager.DriverCommandManager;
 import edu.kis.powp.jobs2d.command.manager.LoggerCommandChangeObserver;
+import edu.kis.powp.jobs2d.drivers.DriverManager;
 
-public class CommandsFeature {
+import javax.swing.*;
 
-	private static DriverCommandManager commandManager;
+public class CommandsFeature implements FeatureInterface {
 
-	public static void setupCommandManager() {
-		commandManager = new DriverCommandManager();
-		commandManager.setDriverManager(DriverFeature.getDriverManager());
-		LoggerCommandChangeObserver loggerObserver = new LoggerCommandChangeObserver();
-		commandManager.getChangePublisher().addSubscriber(loggerObserver);
-	}
+    private static DriverCommandManager commandManager;
+
+    /**
+     * Setup Driver Command Manager.
+     * Parameters required by implemented interface, not used here.
+     *
+     * @param application Application context.
+     * @param freePanel   Free application panel.
+     * @param drvMgr      Driver manager object.
+     */
+    @Override
+    public void setup(Application application, JPanel freePanel, DriverManager drvMgr) {
+        commandManager = new DriverCommandManager();
+        commandManager.setDriverManager(DriverFeature.getDriverManager());
+        LoggerCommandChangeObserver loggerObserver = new LoggerCommandChangeObserver();
+        commandManager.getChangePublisher().addSubscriber(loggerObserver);
+    }
+
+    /**
+     * Get manager of application driver command.
+     *
+     * @return plotterCommandManager.
+     */
+    public static DriverCommandManager getDriverCommandManager() {
+        return commandManager;
```

Comment:
Originally code uses tabs (not spaces) in indent formatting. It makes tracking actual changes more difficult. 
Please fix it!